config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/tasks/zip/beans/Bundle.java

```diff
@@ -22,6 +22,8 @@
     private final Map<String, PolicyBackedService> policyBackedServices = new HashMap<>();
     private final Map<String, IdentityProvider> identityProviders = new HashMap<>();
     private final Map<String, ListenPort> listenPorts = new HashMap<>();
+    private final Map<String, TrustedCert> trustedCerts = new HashMap<>();
+    private final Map<String, String> certificateFiles = new HashMap<>();
```

Comment:
I did not understand why you needed this extra map here, can't you refer the certificate file by name in the `TrustedCert` and then you load the file from the certificates folder?

Reply:
Originally I had a map with the name of the file, to the directory path containing the file. But you may have a point here, I'll see if it's possible to omit this certificateFiles map

Reply:
check how services and policies are done, the service yaml file refer to the path of the policy xml file, and when the bundle is built, the service builder find the policy file and attach to the bundle.